.github/workflows/build.yml

```diff
@@ -9,6 +9,7 @@ on:
 
 env:
   PKG_NAME: "boundary"
+  EDITION: oss
```

Comment:
I think to avoid a conflict on this file, we should derive this variable from something already in the repo. Lets chat to figure out the best way to do that.

Reply:
This configuration must not live in code.  We may want to use GitHub environment secrets for this value.

Reply:
It depends on what you consider "in code". Using GitHub environment secrets might be a good solution for now. But if we eventually want to support using these pre-built ui assets locally, we will need to solve this issue again, and it would be nice if locally it would just "do the right thing" based on something in the repository.

Reply:
We can have a default edition, but it must also be configurable outside of the code in order to support arbitrary editions.

Reply:
I will start a thread in Slack.

Reply:
As decide on slack thread:
- We will retrieve the edition using Go.
- We will add a make script `make edition` to run the Go code mentioned above.
- We will update `build.yaml` to use the make script ratther than `ENV` variable.